curvefs/src/mds/schedule/operatorController.h

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2021 NetEase Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+/*
+ * @Project: curve
+ * @Date: 2021-11-8 11:01:48
+ * @Author: chenwei
+ */
+
+#ifndef CURVEFS_SRC_MDS_SCHEDULE_OPERATORCONTROLLER_H_
+#define CURVEFS_SRC_MDS_SCHEDULE_OPERATORCONTROLLER_H_
+
+#include "curvefs/src/mds/schedule/topoAdapter.h"
+#include "curvefs/src/mds/topology/topology.h"
+#include "src/mds/schedule/operatorControllerTemplate.h"
+
+namespace curvefs {
+namespace mds {
+namespace schedule {
+using OperatorController = curve::mds::schedule::OperatorControllerT<
+    MetaServerIdType, CopySetInfo, CopySetConf, Topology,
+    ::curvefs::mds::topology::CopySetInfo>;
```

Comment:
ditto

Reply:
Refactoring is more convenient in this way, no need to modify the previous code.